src/tag-appender.js

```diff
@@ -75,7 +77,11 @@ function append(file, crossorigin) {
 }
 
 function onCatch(err, optional = false) {
-  console.error('filesAppender failed to load ' + err);
+  ModuleRegistry.notifyListeners('ReactModuleContainerError', {
```

Comment:
Not sure I agree with this, if something is marked as optional, failure to load it shouldn't trigger any error event, and if it's mandatory, the error object propagates out to the component rejection.

To put it another way, I don't want my sentry logs to be spammed with something I know happens and I consider normal.

Reply:
Just because some file is marked as optional doesn't mean it's normal not to load it, only that it's not crucial and shouldn't block the entire application loading.
I think we should report it